homeassistant/components/switch/insteon_plm.py

```diff
@@ -0,0 +1,101 @@
+"""
+Support for INSTEON dimmers via PowerLinc Modem.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/insteon_plm/
+"""
+import logging
+import asyncio
+
+from homeassistant.components.switch import (SwitchDevice)
+from homeassistant.loader import get_component
+
+DEPENDENCIES = ['insteon_plm']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@asyncio.coroutine
+def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
+    """Set up the INSTEON PLM device class for the hass platform."""
+    _LOGGER.info('Provisioning Insteon PLM Switches')
```

Comment:
Thanks for all the feedback, I incorporated your other suggestions, but this one is puzzling me.  I don't really understand what you're asking.  Each of the three platforms has an equivalent of this log line, but they are differentiated (specifically referencing "Switches" and "Lights" and "Binary Sensors."